cmd/ingress-operator/main.go

```diff
@@ -122,6 +138,7 @@ func createDNSManager(cl client.Client, operatorConfig operatorconfig.Config, in
 			AccessID:  string(awsCreds.Data["aws_access_key_id"]),
 			AccessKey: string(awsCreds.Data["aws_secret_access_key"]),
 			DNS:       dnsConfig,
+			Region:    installConfig.Platform.AWS.Region,
```

Comment:
We get the cloud provider from infraConfig and the aws region from installConfig. Is it possible to get the cloud provider from the installConfig too and do away with infraConfig?

Reply:
You may be missing some context here — the kube-system/cluster-config-v1 ConfigMap is deprecated, and all its usages have (for the most part) been replaced by versioned public config API (e.g. `configv1.Infrastructure`). In that respect, this PR is actually a regression as we had already finished the API migration and now we're once again using the deprecated stuff (because ingress didn't need region in the public API until today when metadata became off-limits).

So, when region lands in the public API, we need a followup to revert this PR and switch to the new public API version.